plugins/readtimer.koplugin/main.lua

```diff
@@ -96,19 +93,18 @@ function ReadTimer:addToMainMenu(menu_items)
                                 local hr_str = ""
                                 local min_str = ""
                                 local hr = math.floor(seconds/3600)
-                                if hr == 1 then
-                                    hr_str = string.format("1 %s", hour_string)
-                                elseif hr > 1 then
-                                    hr_str = string.format("%d %s", hr, hours_string)
+                                if hr > 0 then
+                                    hr_str = T(N_("1 hour", "%1 hours", hr), hr)
                                 end
                                 local min = math.floor((seconds%3600)/60)
-                                if min == 1 then
-                                    min_str = string.format("1 %s", minute_string)
-                                elseif min > 1 then
-                                    min_str = string.format("%d %s", min, minutes_string)
+                                if min > 0 then
+                                    min_str = T(N_("1 minute", "%1 minutes", min), min)
+                                    if hr_str ~= "" then
+                                        hr_str = hr_str .. " "
+                                    end
                                 end
                                 UIManager:show(InfoMessage:new{
-                                    text = T(_("Timer set to: %1:%2\nIt's %3 %4 from now"),
+                                    text = T(_("Timer set to: %1:%2.\nIt's %3%4 from now."),
```

Comment:
That "it's" is a bit weird. Maybe something like this.
```suggestion
                                    text = T(_("Timer set to: %1:%2.\n\nThat's %3%4 from now."),
```